content/docs/principles/prototyping-high-risk-items-first.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Prototyping high risk items first
+weight: 1
```

Comment:
I don't think this should be the first. Let's keep this one after "Focus on Fundamentals".